src/content/docs/en/guides/content-collections.mdx

```diff
@@ -144,6 +144,8 @@ To define your first collection, create a `src/content/config.ts` file if one do
 2. **Define each collection that you'd like to validate.** This includes a `type` (introduced in Astro v2.5.0) specifying whether the collection contains content authoring formats like Markdown (`type: 'content'`) or data formats like JSON or YAML (`type: 'data'`). It also includes a `schema` that defines the shape of your frontmatter or entry data.
 3. **Export a single `collections` object** to register your collections.
 
+For example, your `content.ts` file could look something like this:
+
```

Comment:
I don't know why this was added? It is not appropriate in this PR.

Reply:
Reverted